packages/uui-toast-notification/lib/uui-toast-notification.test.ts

```diff
@@ -215,19 +215,20 @@ describe('UUIToastNotificationElement', () => {
 
     describe('autoClose can be paused after open-animation', () => {
       it('did element close wait', async () => {
-        element.autoClose = 200;
+        element.autoClose = 210;
         element.open = true;
 
-        await sleep(600); // enough time for opening animation to be done.
+        await sleep(200); // enough time for opening animation to be done.
```

Comment:
This test is about testing that we can pause the autoclose-timer after the open-animation is done. Therefore the time here needs to be above the amount of time the open-animation takes (including time to get started).

So that is why I originally went with 600. Animation is set to 480ms but we need to add a little bit extra for initialization and rendering. So that's why I picked 600ms.

This PR is changing the times in quite a few places in this PR. Some of the places I guess will work well. But some places I´m affrighted that's ruining the purpose of the tests?